code/game/objects/items/weapons/implants/implant.dm

```diff
@@ -294,7 +294,10 @@ the implant may become unstable and either pre-maturely inject the subject or si
 	if(!iscarbon(M))
 		return 0
 	var/mob/living/carbon/H = M
-	if(isrevhead(H) || iscultist(H))
+	if(isrevhead(H))
+		H.visible_message("<span class='big danger'>[H] seems to resist the implant!</span>", "<span class='danger'>You feel the corporate tendrils of Nanotrasen try to invade your mind!</span>")
```

Comment:
you don't necessarily have to copypaste this line, you could just `isrevhead(H) || (iscultist(H) && veil_thickness >= CULT_ACT_II)`